### DISCUSSION TITLE
Satisfaction survey questions

### DISCUSSION THREAD

**Discussion by shawnthompson** [gc-da11yn/gc-da11yn.github.io#118] [Best Practices (Open Discussions)]
Would like to do user testing and create a best practice when doing a satisfaction question in an HTML form.

![image](https://user-images.githubusercontent.com/2575266/147127602-159b5890-1727-4aea-8c36-a0da08d60899.png)

**Image caption:** A rating system of numbers from 0 to 10, where 0 is equal to **Worst Off** and 10 being equal to **Best Off**.

These forms of questions can create barriers for users with cognitive disabilities and aren't always clear. 

It would be nice to have a standard within the Government of Canada on how to format these forms of questions online.

Alternative could be:
* `<select>` / dropdown, giving clear labeling
* radio buttons also with clear labelling for each
* `input[type="range"]`
* using stars / icons (what if we used maple leafs as a rating system through the Canada.ca?)

I think usability test would be needed and I'd like to see us report our findings on the site.

[Slack thread](https://web-a11y.slack.com/archives/C042TSFGN/p1615987221031400) on topic.

If you aren't part of the web-a11y slack workspace, let me know and I'll send you an invitation.

**Comment by duboisp** [comment]
What about to create a component that will transform the rating interface to allow the user to choose an input format that best fit his need?
